src/scalper.ts

```diff
@@ -601,32 +601,74 @@ export class Scalper {
     const stdDevSpread = this.impliedVol / Math.sqrt((365 * 24 * 60 * 60) / scalperWindow) * zScore;
     const slippageTolerance = Math.min(stdDevSpread/ 2, slippageMax.get(this.symbol));
     const deltaThreshold = Math.max(dipTotalGamma * stdDevSpread * fairValue * gammaThreshold, this.minSpotSize);
+    let hedgePrice = hedgeDeltaTotal < 0 ? fairValue * (1 + slippageTolerance) : fairValue * (1 - slippageTolerance);
     if (Math.abs(hedgeDeltaTotal * fairValue) < (deltaThreshold * fairValue)) {
       console.log(this.symbol, "Delta Netural <", deltaThreshold);
       return;
     } else {
-      console.log(this.symbol, "Above delta threshold:", hedgeDeltaTotal, ">", deltaThreshold);
+      // Adjust delta for slippage allowed
+      const slipapgeDelta = getDIPDelta(dipProduct, hedgePrice, this.symbol);
+      const dipDeltaDiff = slipapgeDelta - dipTotalDelta;
+      hedgeDeltaTotal = hedgeDeltaTotal + dipDeltaDiff;
```

Comment:
in places like this, use +=
it is easier to read and more clear what is going on